packages/tailwind-config/index.js

```diff
@@ -0,0 +1,149 @@
+import tailwindCssAnimatePlugin from 'tailwindcss-animate';
```

Comment:
Nothing new here — this file is just moved over to this new package from `packages/ui`.